support/client/lib/vwf/model/kineticjs.js

```diff
@@ -2153,8 +2153,11 @@ define( [ "module",
                     kineticObj.scale( { "x": height / imageObj.height ,"y": height / imageObj.height } );
                 }
             } else {
-                kineticObj.width( imageObj.width );
-                kineticObj.height( imageObj.height );
+                var kImg = kineticObj.image();
+                if ( ( kImg.width !== imageObj.width ) || ( kImg.height !== imageObj.height ) ) {
+                    kineticObj.width( imageObj.width );
+                    kineticObj.height( imageObj.height );
+                }
```

Comment:
It looks like it might make sense to split these into two conditionals:  Update the width if the width changed, and update the height if the height changed.